jqassistant.properties

```diff
@@ -1,15 +1,21 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.9.0,1.10.0,1.11.0
-publicVersions=1.12.0
+archivedVersions=1.9.0,1.10.0,1.11.0,1.12.0
+publicVersions=1.12.1
 
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+1.12.1.description=Improved sensor performance and ensure that all issues that cannot be attached to a file are displayed on project level
+1.12.1.date=2023-03-16
+1.12.1.sqVersions=[8.9,LATEST]
+1.12.1.changelogUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin/blob/master/release-notes.adoc#1121
+1.12.1.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/contrib/sonarqube/sonar-jqassistant-plugin/1.12.1/sonar-jqassistant-plugin-1.12.1.jar
+
 1.12.0.description=Added support for reporting warnings generated by jQAssistant as issues in SonarQube
 1.12.0.date=2022-07-13
-1.12.0.sqVersions=[8.9,LATEST]
+1.12.0.sqVersions=[8.9,9.5.*]
```

Comment:
Since previous compatibility was with `LATEST`, the range needs to end with the current version. Otherwise your users on 9.6, 9.7, 9.8 & 9.9 won't be prompted to upgrade.

Reply:
So it shall be `9.9.*`? Fixing it. Thanks for the explanation!